res/css/views/dialogs/_SpotlightDialog.pcss

```diff
@@ -97,7 +97,11 @@ limitations under the License.
             }
 
             &.mx_SpotlightDialog_filterPublicRooms::before {
-                mask-image: url("$(res)/img/element-icons/roomlist/explore.svg");
+                mask-image: url("$(res)/img/element-icons/roomlist/hash-circle.svg");
+            }
+
+            &.mx_SpotlightDialog_filterPublicSpaces::before {
+                mask-image: url("$(res)/img/element-icons/spaces.svg");
             }
```

Comment:
I believe we're trying to favour using SVG react elements over `mask-image` 
https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/icons.md#icons

Reply:
Agreed but I think matching the surrounding code is more important given all these icons will need changing as part of the compound work anyway